popup.js

```diff
@@ -83,11 +83,47 @@ document.addEventListener('DOMContentLoaded', function () { // this function  st
                 console.log("user has deleted the board manually.");
                 board_create_function();
             });
-
+        
+        // Populating Fields
         chrome.tabs.getSelected(null, function (tab) {
             document.getElementById('data_url').value = tab.url;
         });
 
+        function getFieldsFromDOM() {
+            var fields = []
+            //Account for Companies which have no URL/Link
+            try {
+                var url = document.getElementsByClassName("jobs-details-top-card__company-url")[0].innerText.trim();
```

Comment:
Can you convert these variables to `let` instead and see if it works fine? Usually, it's best practice to avoid any vars :)

Reply:
No problem!